docs/CONTRIBUTING.md

```diff
@@ -1,45 +1,57 @@
-# Guide to Contributors
+# Contributor Guide
 
-Hello and thank you for your interest in working on Flix!
+Hello and thank you for your interest in working on Flix! :-)
 
-The purpose of this document is to help you get started.
+## The Process
 
-## The Standard Process
+We suggest that you proceed by following these steps:
 
-We don't actually have a standardized process, but usually the following steps happen:
+1. Pick an issue on GitHub that you are interested in. 
+   - It could be adding a new feature, fixing a bug in the compiler, and so forth.
 
-1. You pick an issue on GitHub that you are interested in. 
-
-    It could be adding a new feature, fixing a bug in the compiler, and so forth.
-
-2. You express interest in the issue on GitHub.
-
-    We try to ensure that multiple people are not working on the same issue -- at least not without being aware of it.
+2. Express interest in the issue on GitHub.
+    - We try to ensure that multiple people are not working on the same issue.
  
 3. We discuss the issue and how best to fix it.
-
-    The idea is to come up with an implementation plan and to hash out any design choices.
+    - The idea is to come up with an implementation plan and to hash out any design choices.
 
 4. You fork the Flix repository to your own account on GitHub.
 
-5. You clone this repository to your local machine.
+5. You clone the forked repository to your own machine.
 
 6. You make the required changes, add test cases, documentation, and so forth.
 
 7. You push the local changes to your GitHub repository and create a new pull request (PR).
-   - See [Pull Requests](PULLREQUEST.md) for information on formatting
```

Comment:
Does GitHub treat that file specially? E.g. displaying it when someone opens a PR the first time?

Reply:
No, there is no support for that.